Nil pointer dereference while previewing PR and Issues with empty body

### Describe the bug

PR and issues with an empty body are failing on preview. 

#### gh version
gh version 0.5.4 (2020-02-04)
https://github.com/cli/cli/releases/tag/v0.5.4

### Steps to reproduce the behavior
#### PR

```
1. Issue command
> gh pr view -p 2 -R arkentos/tableturner

2.  Output:

This is a test PR with empty body
arkentos wants to merge 1 commit into master from README

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x156c8ea]

goroutine 1 [running]:
github.com/vilmibm/go-termd.(*Compiler).visit(0xc000633a40, 0x0, 0x0, 0x203000)
	/home/runner/go/pkg/mod/github.com/vilmibm/go-termd@v0.0.4/termd.go:51 +0x3a
github.com/vilmibm/go-termd.(*Compiler).visit(0xc000633a40, 0xc00036a140, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/vilmibm/go-termd@v0.0.4/termd.go:53 +0x305
github.com/vilmibm/go-termd.(*Compiler).Compile(0xc000633a40, 0x0, 0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/vilmibm/go-termd@v0.0.4/termd.go:46 +0xb2
github.com/cli/cli/utils.RenderMarkdown(0x0, 0x0, 0x0, 0x0)
	/home/runner/work/cli/cli/utils/utils.go:46 +0x3cd
github.com/cli/cli/command.printPrPreview(0x1845140, 0xc0000a4008, 0xc0004f41c0)
	/home/runner/work/cli/cli/command/pr.go:324 +0x32d
github.com/cli/cli/command.prView(0x1c9b140, 0xc000031f40, 0x1, 0x4, 0x0, 0x0)
	/home/runner/work/cli/cli/command/pr.go:306 +0x1d6
github.com/spf13/cobra.(*Command).execute(0x1c9b140, 0xc000031f00, 0x4, 0x4, 0x1c9b140, 0xc000031f00)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0x1c99840, 0x5, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
main.main()
	/home/runner/work/cli/cli/cmd/gh/main.go:33 +0xde
```

#### Issue

```
1. Issue command
> gh issue view -p 1 -R arkentos/tableturner

2.  Output:

This is a test issue with empty body
opened by arkentos. 0 comments.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x156c8ea]

goroutine 1 [running]:
github.com/vilmibm/go-termd.(*Compiler).visit(0xc0006dd998, 0x0, 0xa00000000, 0x2000)
	/home/runner/go/pkg/mod/github.com/vilmibm/go-termd@v0.0.4/termd.go:51 +0x3a
github.com/vilmibm/go-termd.(*Compiler).visit(0xc0006dd998, 0xc00038a000, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/vilmibm/go-termd@v0.0.4/termd.go:53 +0x305
github.com/vilmibm/go-termd.(*Compiler).Compile(0xc0006dd998, 0x0, 0x0, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/vilmibm/go-termd@v0.0.4/termd.go:46 +0xb2
github.com/cli/cli/utils.RenderMarkdown(0x0, 0x0, 0x0, 0x0)
	/home/runner/work/cli/cli/utils/utils.go:46 +0x3cd
github.com/cli/cli/command.printIssuePreview(0x1845140, 0xc0000a8008, 0xc0005563c0)
	/home/runner/work/cli/cli/command/issue.go:258 +0x3a0
github.com/cli/cli/command.issueView(0x1c9a740, 0xc0004ad900, 0x1, 0x4, 0x0, 0x0)
	/home/runner/work/cli/cli/command/issue.go:235 +0x1cb
github.com/spf13/cobra.(*Command).execute(0x1c9a740, 0xc0004ad8c0, 0x4, 0x4, 0x1c9a740, 0xc0004ad8c0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0x1c99840, 0x5, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
main.main()
	/home/runner/work/cli/cli/cmd/gh/main.go:33 +0xde
```


### Expected vs actual behavior

#### Expected behavior for PR
```
This is a test PR with empty body
arkentos wants to merge 1 commit into master from README
View this pull request on GitHub: https://github.com/arkentos/tableturner/pull/2
```

#### Expected behavior for Issue
```
This is a test issue with empty body
opened by arkentos. 0 comments.
View this issue on GitHub: https://github.com/arkentos/tableturner/issues/1
```

Actual behavior is the nil pointer dereference error

### Logs
Refer above for the output of PR and Issues

--- a/command/issue.go
+++ b/command/issue.go
@@ -255,9 +255,11 @@ func printIssuePreview(out io.Writer, issue *api.Issue) {
 		utils.Pluralize(issue.Comments.TotalCount, "comment"),
 		coloredLabels,
 	)))
-	fmt.Fprintln(out)
-	fmt.Fprintln(out, utils.RenderMarkdown(issue.Body))
-	fmt.Fprintln(out)
+	if issue.Body != "" {
+		fmt.Fprintln(out)
+		fmt.Fprintln(out, utils.RenderMarkdown(issue.Body))
+		fmt.Fprintln(out)
+	}
 	fmt.Fprintf(out, utils.Gray("View this issue on GitHub: %s\n"), issue.URL)
 }
 

--- a/command/issue_test.go
+++ b/command/issue_test.go
@@ -288,6 +288,51 @@ func TestIssueView_preview(t *testing.T) {
 	}
 }
 
+func TestIssueView_previewWithEmptyBody(t *testing.T) {
+	initBlankContext("OWNER/REPO", "master")
+	http := initFakeHTTP()
+	http.StubRepoResponse("OWNER", "REPO")
+
+	http.StubResponse(200, bytes.NewBufferString(`
+		{ "data": { "repository": { "hasIssuesEnabled": true, "issue": {
+		"number": 123,
+		"body": "",
+		"title": "ix of coins",
+		"author": {
+			"login": "marseilles"
+		},
+		"labels": {
+			"nodes": [
+				{"name": "tarot"}
+			]
+		},
+		"comments": {
+		  "totalCount": 9
+		},
+		"url": "https://github.com/OWNER/REPO/issues/123"
+	} } } }
+	`))
+
+	output, err := RunCommand(issueViewCmd, "issue view -p 123")
+	if err != nil {
+		t.Errorf("error running command `issue view`: %v", err)
+	}
+
+	eq(t, output.Stderr(), "")
+
+	expectedLines := []*regexp.Regexp{
+		regexp.MustCompile(`ix of coins`),
+		regexp.MustCompile(`opened by marseilles. 9 comments. \(tarot\)`),
+		regexp.MustCompile(`View this issue on GitHub: https://github.com/OWNER/REPO/issues/123`),
+	}
+	for _, r := range expectedLines {
+		if !r.MatchString(output.String()) {
+			t.Errorf("output did not match regexp /%s/\n> output\n%s\n", r, output)
+			return
+		}
+	}
+}
+
 func TestIssueView_notFound(t *testing.T) {
 	initBlankContext("OWNER/REPO", "master")
 	http := initFakeHTTP()

--- a/command/pr.go
+++ b/command/pr.go
@@ -308,9 +308,11 @@ func printPrPreview(out io.Writer, pr *api.PullRequest) {
 		pr.BaseRefName,
 		pr.HeadRefName,
 	)))
-	fmt.Fprintln(out)
-	fmt.Fprintln(out, utils.RenderMarkdown(pr.Body))
-	fmt.Fprintln(out)
+	if pr.Body != "" {
+		fmt.Fprintln(out)
+		fmt.Fprintln(out, utils.RenderMarkdown(pr.Body))
+		fmt.Fprintln(out)
+	}
 	fmt.Fprintf(out, utils.Gray("View this pull request on GitHub: %s\n"), pr.URL)
 }
 

--- a/command/pr_test.go
+++ b/command/pr_test.go
@@ -313,6 +313,40 @@ func TestPRView_previewCurrentBranch(t *testing.T) {
 	}
 }
 
+func TestPRView_previewCurrentBranchWithEmptyBody(t *testing.T) {
+	initBlankContext("OWNER/REPO", "blueberries")
+	http := initFakeHTTP()
+	http.StubRepoResponse("OWNER", "REPO")
+
+	jsonFile, _ := os.Open("../test/fixtures/prView_EmptyBody.json")
+	defer jsonFile.Close()
+	http.StubResponse(200, jsonFile)
+
+	restoreCmd := utils.SetPrepareCmd(func(cmd *exec.Cmd) utils.Runnable {
+		return &outputStub{}
+	})
+	defer restoreCmd()
+
+	output, err := RunCommand(prViewCmd, "pr view -p")
+	if err != nil {
+		t.Errorf("error running command `pr view`: %v", err)
+	}
+
+	eq(t, output.Stderr(), "")
+
+	expectedLines := []*regexp.Regexp{
+		regexp.MustCompile(`Blueberries are a good fruit`),
+		regexp.MustCompile(`nobody wants to merge 8 commits into master from blueberries`),
+		regexp.MustCompile(`View this pull request on GitHub: https://github.com/OWNER/REPO/pull/10`),
+	}
+	for _, r := range expectedLines {
+		if !r.MatchString(output.String()) {
+			t.Errorf("output did not match regexp /%s/\n> output\n%s\n", r, output)
+			return
+		}
+	}
+}
+
 func TestPRView_currentBranch(t *testing.T) {
 	initBlankContext("OWNER/REPO", "blueberries")
 	http := initFakeHTTP()

--- a/test/fixtures/prView_EmptyBody.json
+++ b/test/fixtures/prView_EmptyBody.json
@@ -1,0 +1,46 @@
+{
+  "data": {
+    "repository": {
+      "pullRequests": {
+        "nodes": [
+          {
+            "number": 12,
+            "title": "Blueberries are from a fork",
+            "body": "yeah",
+            "url": "https://github.com/OWNER/REPO/pull/12",
+            "headRefName": "blueberries",
+            "baseRefName": "master",
+            "headRepositoryOwner": {
+              "login": "hubot"
+            },
+            "commits": {
+              "totalCount": 12
+            },
+            "author": {
+              "login": "nobody"
+            },
+            "isCrossRepository": true
+          },
+          {
+            "number": 10,
+            "title": "Blueberries are a good fruit",
+            "body": "",
+            "url": "https://github.com/OWNER/REPO/pull/10",
+            "baseRefName": "master",
+            "headRefName": "blueberries",
+            "author": {
+              "login": "nobody"
+            },
+            "headRepositoryOwner": {
+              "login": "OWNER"
+            },
+            "commits": {
+              "totalCount": 8
+            },
+            "isCrossRepository": false
+          }
+        ]
+      }
+    }
+  }
+}

